### PR TITLE
Enhance the README file of the BitBswup project (#1449)

### LINKED ISSUE #1449: Enhance the README file of the `BitBswup` project
the first version of the README.md file of the BitBswup project has some shortcomings such as detailed steps and also some other required settings for the final project.

### PR DESCRIPTION
**Pull request by Cyrus-Sushiant**
This closes #1449

### CODE REVIEW

File: src/Tooling/Bit.Tooling.Bswup/README.md [hunk 1]
```diff
@@ -1,7 +1,90 @@
...
 
-1- install the `Bit.Tooling.Bswup` nuget package
-2- add the requied references to the default document based on the sample in the `index.html` file of the demo project in this repo
-3- configure additional settings in the service worker file (based on the sample shown in the `service-worker.js` file of the demo project)
\ No newline at end of file
+1.Install the `Bit.Tooling.Bswup` nuget package
+2. Disable static file caching.You can follow below code in `Startup.cs` file
+```csharp
+
+public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
```
Comment (ysmoradi) [inline src/Tooling/Bit.Tooling.Bswup/README.md:9]: Please remove Configure method. app.UseStaticFiles is enough. They might be using new .net 6 style project which uses web app builder

**Review by ysmoradi — CHANGES_REQUESTED** [review]

File: src/Tooling/Bit.Tooling.Bswup/README.md [hunk 2]
```diff
@@ -1,7 +1,90 @@
...
+
+<script src="_content/Bit.Tooling.Bswup/bit-bswup.js"
+            scope="/"
+            log="verbose"
+            sw="service-worker.js"
+            handler="bitBswupHandler"></script>
+
+```
+
+- scope: The scope of the service worker determines which files the service worker controls.
```
Comment (ysmoradi) [inline src/Tooling/Bit.Tooling.Bswup/README.md:46]: Provide a link to scope in service worker docs

File: src/Tooling/Bit.Tooling.Bswup/README.md [hunk 3]
```diff
@@ -1,7 +1,90 @@
...
+- log: The log level for log provider. log options: `info`, `verbose`, `debug`, `error`
+-sw: The sw is name and path service worker file.
```
Comment (ysmoradi) [inline src/Tooling/Bit.Tooling.Bswup/README.md:48]: space is missing before sw

File: src/Tooling/Bit.Tooling.Bswup/README.md [hunk 4]
```diff
@@ -1,7 +1,90 @@
...
+        case 'updatefound':
+            return console.log('new version is downloading...');
+        case 'statechange':
+            return console.log('new version state has changed to:', data.currentTarget.state);
+        case 'controllerchange':
+            return console.log('sw controller changed:', data);
+        case 'installing':
+            return console.log('installing new version:', data.version);
+        case 'installed':
+            reloadButton.style.display = 'block';
```
Comment (ysmoradi) [inline src/Tooling/Bit.Tooling.Bswup/README.md:67]: what's reloadButton here?

File: src/Tooling/Bit.Tooling.Bswup/README.md [hunk 5]
```diff
@@ -2,53 +2,50 @@
...
+4. In the default document (`index.html` or `_Host.cshtml`), add an `autostart = "false"` attribute and value to the <script> tag for the Blazor script.
 
 ```html
 
 <script src="_framework/blazor.webassembly.js" autostart="false"></script>
 
 ```
 
-5. Add the `Bit.Tooling.Bswup` reference after the <script> tag for the Blazor script.
+5. In the default document (`index.html` or `_Host.cshtml`), add the `Bit.Tooling.Bswup` reference after the <script> tag for the Blazor script.
```
Comment (ysmoradi) [inline src/Tooling/Bit.Tooling.Bswup/README.md:31]: _Host.cshtml, index.html or _Layout.cshtml
This is also wrong in bit component getting started where we add style & script as described in getting started. @maryamhdr

**Review by ysmoradi — CHANGES_REQUESTED** [review]

**Comment by msynk** [comment]
@Cyrus-Sushiant any updates?

**Review by msynk — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**